les/server_handler.go

```diff
@@ -55,6 +55,7 @@ const (
 	MaxTxSend                = 64  // Amount of transactions to be send per request
 	MaxTxStatus              = 256 // Amount of transactions to queried per request
 	MaxEtherbase             = 1
+	MaxGatewayFee            = 1000000 //harcoded temp limit for now @rayyuan
```

Comment:
This should probably be 1 simply because there is no reason for it to be any higher. (Why would someone
 request multiple gateway fee responses?)